labellab-server/test/analytics.js

```diff
@@ -0,0 +1,173 @@
+var expect = require('chai').expect
+var mongoose = require('mongoose')
+let chai = require('chai')
+let chaiHttp = require('chai-http')
+
+var Project = require('../models/project')
+var User = require('../models/user')
+var Label = require('../models/label')
+let server = require('../app')
+let conn = require('../config/test')
+const Info = require('./info')
+
+const userInfo = {
+	name: 'name',
+	username: 'username',
+	email: 'email@gmail.com',
+	password: 'password',
+	password2: 'password'
+}
+
```

Comment:
Instead of creating a new userInfo object you can use dump data from `info.js`.

Reply:
sure I will do that!